ui/app/AppLayouts/Onboarding2/OnboardingLayout.qml

```diff
@@ -147,7 +166,9 @@ Page {
         syncState: root.onboardingStore.syncState
         addKeyPairState: root.onboardingStore.addKeyPairState
 
-        seedWords: root.onboardingStore.getMnemonic().split(" ")
+        getSeedWords: function () {
```

Comment:
Why is this a function now?

Reply:
We can only get the mnemonic when the keycard is authorized or initialized, so I pass the function to be called later when we have the right to call it.